mlflow/tracing/clients/local.py

```diff
@@ -93,6 +65,7 @@ def get_trace(self, request_id: str) -> Optional[Trace]:
         with self._lock:
             for trace in self.queue:
                 if trace.trace_info.request_id == request_id:
+                    get_display_handler().display_traces([trace])
```

Comment:
Can we move this call to upstream i.e. the user-facing APIs (`mlflow.MlflowClient().get_trace()`, `mlflow.get_traces`), and make this in-memory client focus purely on *getting* traces without side-effect? These getter APIs are also used internally to access to in-memory trace like [here](https://github.com/mlflow/mlflow/blob/tracing/mlflow/tracking/client.py#L624).

Reply:
makes sense, i'll update!